Tasks/AzureCLIV1/task.json

```diff
@@ -97,7 +97,7 @@
             "label": "Access service principal details in script",
             "defaultValue": "false",
             "required": false,
-            "helpMarkDown": "Adds service principal id and key of the Azure endpoint you chose to the script's execution environment. You can use these variables: `$servicePrincipalId` and `$servicePrincipalKey` in your script",
+            "helpMarkDown": "Adds service principal id and key of the Azure endpoint you chose to the script's execution environment. You can use these variables: `$servicePrincipalId` and `$servicePrincipalKey` in your script.\n\nPS: Valid only with ARM Endpoint service connection with 'Service Principal' authentication scheme.",
```

Comment:
No need to put PS. You can say "This is honored only when the Azure endpoint has Service Principal authentication scheme."

Reply:
Done